libs/barista-components/table/src/row.scss

```diff
@@ -24,10 +28,17 @@
 }
 
 :host-context(.dt-table-interactive-rows):hover {
-  border: solid 1px $gray-500;
+  border: solid 1px var(--interactive-border-color);
```

Comment:
Hmm, this was gray-500 before and is now gray-100?
```suggestion
  border: solid 1px var(--interactive-border-color);
```

Reply:
I am not sure how this happened, good catch.